docs/source/Object Oriented Interface.rst

```diff
@@ -7,4 +7,46 @@ Object Oriented Interface
    :members:
 
 
+Terms in the LWA Column Budget
+------------------------------
+
+With the LWA column budget as formulated by `Neal et al. <https://doi.org/10.1029/2021GL097699>`_ (2022, supporting information, equations 5-7), the output fields of :py:meth:`QGField.compute_lwa_and_barotropic_fluxes` correspond to terms as follows:
+
+.. math::
+
+   \frac{\partial}{\partial t} \boxed{ \langle A \rangle \cos(\phi) }_\mathtt{~lwa\_baro} ~ = ~
+        & \boxed{ - \frac{1}{a \cos(\phi)} \frac{\partial}{\partial \lambda} \langle F_{\lambda} \rangle }_\mathtt{~convergence\_zonal\_advective\_flux} \\
+        & \boxed{ - \frac{1}{a \cos(\phi)} \frac{\partial}{\partial \phi'} \langle F_{\phi'} \cos(\phi + \phi') \rangle }_\mathtt{~divergence\_eddy\_momentum\_flux} \\
```

Comment:
To stick to the notations used in the Science paper, I'll substitute the expression of `` here. May you change this line to:
```
& \boxed{ + \frac{1}{a \cos(\phi)} \frac{\partial}{\partial \phi'} \langle u_e v_e \cos^2(\phi + \phi') \rangle }_\mathtt{~divergence\_eddy\_momentum\_flux} \\

```